include/sys/dmu.h

```diff
@@ -381,6 +381,7 @@ typedef struct dmu_buf {
 #define	DMU_POOL_CREATION_VERSION	"creation_version"
 #define	DMU_POOL_SCAN			"scan"
 #define	DMU_POOL_ERRORSCRUB		"error_scrub"
+#define	DMU_POOL_LAST_SCRUBBED_TXG	"last_scrubbed_txg"
```

Comment:
It's tempting to store a last scrubbed time while we're here.  This information is stored in the `scn->scn_phys.scn_end_time` but that will be lost if a resilver occurs after the scrub completes.  That said, I'm not sure where we'd report this to the user so I'm fine not including it in this initial version.

Reply:
I can look into this as a separate PR.